api/src/main/resources/messages.properties

```diff
@@ -74,7 +74,7 @@ adminui.myAccount=My Account
 adminui.myAccount.password.label=Password
 adminui.myAccount.changeSecretQuestion.label=Secret Question
 adminui.myAccount.defaults.label=User Defaults
-adminui.myAccount.defaultSettings.label=Default Settings
+adminui.myAccount.defaultSettings.label=My Languages
```

Comment:
I think you should rename the key too and make sure all its usages are updated if any.